tracker/cli/setup.py

```diff
@@ -0,0 +1,181 @@
+from click import echo, option, pass_context, password_option, Choice, BadParameter
+from re import match, IGNORECASE
+from pathlib import Path
+from os.path import exists
+from os import rename
+
+from config import basedir, TRACKER_PASSWORD_LENGTH_MIN, TRACKER_PASSWORD_LENGTH_MAX
+from tracker.model.user import User, username_regex
+from tracker.model.enum import UserRole
+from . import cli
+
+
+@cli.group()
+def setup():
+    """Setup and bootstrap the application."""
+    pass
+
+
+@setup.command()
+@option('--purge', is_flag=True, help='Purge all data and tables.')
+@pass_context
+def bootstrap(ctx, purge=False):
+    """Bootstrap the environment.
+
+    Create all folders, database tables and other things that are required to
+    run the application.
+
+    An initial administrator user must be created separately."""
```

Comment:
I don't know how pedantic we are with this, but some people ask the `"""` to be in their own lines...

Reply:
no clue, i don't care much and even python stdlib has a mix of both :yum: